tokio-rustls/src/server.rs

```diff
@@ -123,3 +126,13 @@ where
         stream.as_mut_pin().poll_shutdown(cx)
     }
 }
+
+#[cfg(unix)]
+impl<IO> AsRawFd for TlsStream<IO>
```

Comment:
Should we also have a windows version?

Reply:
Good call, I was being selfish. Implemented!